source/MaterialXRender/OpenGL/GLTextureHandler.cpp

```diff
@@ -113,11 +113,66 @@ bool GLTextureHandler::bindImage(const string &identifier)
         }
         glActiveTexture(GL_TEXTURE0 + resourceId);
         glBindTexture(GL_TEXTURE_2D, resourceId);
+
+        // Set up texture properties
+        //
+        GLint minFilterType = mapFilterTypeToGL(imageProperties.filterType);
+        GLint magFilterType = (minFilterType == GL_LINEAR_MIPMAP_LINEAR) ? GL_LINEAR : minFilterType;
+        GLint uaddressMode = mapAddressModeToGL(imageProperties.uaddressMode);
+        GLint vaddressMode = mapAddressModeToGL(imageProperties.vaddressMode);
+        
+        glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_WRAP_S, uaddressMode);
+        glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_WRAP_T, vaddressMode);
+        glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MAG_FILTER, magFilterType);
+        glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MIN_FILTER, minFilterType);
+
         return true;
     }
     return false;
 }
 
+int GLTextureHandler::mapAddressModeToGL(const MaterialX::ValuePtr value)
```

Comment:
From the comment above, could use just int instead of ValuePtr here.

Reply:
Yes, Here I can map to int first before calling as that's the assumed input.